development/tools/jobs/tester/buildjob/jobs.go

```diff
@@ -0,0 +1,168 @@
+package buildjob
+
+import (
+	"fmt"
+	. "github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/test-infra/prow/config"
+	"path"
+	"strings"
+	"testing"
+)
+
+type Suite struct {
+	path              string
+	repository        string
+	image             string
+	releases          []*SupportedRelease
+	runIfChanged      string
+	runIfChangedCheck string
+	fileSuffix        string
+	noMaster          bool
+}
+
+
+func NewSuite(opts ...Option) *Suite {
+	suite := &Suite{
+		releases: GetAllKymaReleases(),
```

Comment:
it's also a default, maybe it should be moved to `setDefault` method. The best option is always to have only one approach for setting defaults value.